Add basic support for targets related to wasm32.

TheVeryDarkness opened the pull request:
Map rust target wasm32-unknown-emscripten and wasm32-unknown-unknown to vcpkg triplet wasm32-emscripten.

pkgw commented:
Thanks for submitting! It would be great to also add some test coverage and brief documentation to go along with this support, if you're able.

TheVeryDarkness commented:
Okay, but I'm afraid that I need some time for it. I may be busy for several days as I'm attending a game.

TheVeryDarkness commented:
@pkgw I've added a test for wasm32-unknown-unknown target and added a little documentation. But the workflows just failed the same as it did on Linux and macOS. You can see it in [actions](https://github.com/TheVeryDarkness/vcpkg-rs/actions).

TheVeryDarkness commented:
Sorry the commit about documentation hasn't be pushed to github yet. Bad network.

TheVeryDarkness commented:
> Sorry the commit about documentation hasn't be pushed to github yet. Bad network.

I mean 4edd92f.

TheVeryDarkness commented:
@pkgw Any more suggestions?

pkgw commented:
I'm very sorry, I've fallen far behind on my GitHub activity lately. I've simply punted on most things that were in my queue and am trying to catch back up. This now looks good to me, especially since it is such a non-invasive change. @waych please shout in the next few days if you have any comments here, otherwise I'll go ahead and merge.
